### ISSUE TITLE
Ice Cap Icicle & Skydeck Fence + Platform Texture

### ISSUE THREAD

**Issue by ghost** [SonicFreak94/sadx-dc-lighting#35]
Posting on both pages.

Ice Cap Icicle has different color.
http://screenshotcomparison.com/comparison/199783

Fence seems to ignore lighting (?)
There is a different looking texture on platform.
That ceiling crane thingy has different position of texture (?)
http://screenshotcomparison.com/comparison/199782

**Comment by SonicFreak94** [comment]
This looks like a mix of alpha issues and lack of texture mirroring, which is better suited for the [sadx_dreamcast](https://github.com/PiKeyAr/sadx_dreamcast) repo.

The lack of lighting on those rails could actually be due to incorrect palette selection though, so it's worth investigating. It's also interesting to see the subtle lighting differences overall, which could be a fault in the palette interpolation system.

Thanks for the report.

**Comment by PiKeyAr** [comment]
I think the icicles are like that because of palette selection, while everything else is correctable on my end. The fence does look like it ignores lighting, and the texture mirroring problem can be resolved by reimporting the DC model of that object.

**Comment by PiKeyAr** [comment]
After removing the "ignore light" flag on the rails/fence, I found out that palette selection is also involved. Since some rails are landtable objects, and others are SET objects, they are forced to load different palettes, and lighting looks different in the areas where they are joined together. To fix this, a palette selection method needs to be implemented in the lighting mod, particularly two things:

1) "Dreamcast mode" for landtable objects, which is basically what I requested in #20 
2) A way to set palette indices manually for certain objects/models/materials/whatever. I'd be willing to go through all the objects/materials and adjust them manually when this is implemented.
